JetNews/app/src/main/java/com/example/jetnews/ui/home/HomeScreens.kt

```diff
@@ -112,21 +126,17 @@ fun HomeFeedWithArticleDetailsScreen(
         openDrawer = openDrawer,
         navigateToInterests = navigateToInterests,
         homeListLazyListState = homeListLazyListState,
-        scaffoldState = scaffoldState
-    ) { hasPostsUiState, modifier ->
-        Row {
+        scaffoldState = scaffoldState,
+        modifier = modifier,
+    ) { hasPostsUiState, contentModifier ->
+        Row(Modifier.padding(top = 8.dp)) {
```

Comment:
This top padding looks a little weird when scroll, since the content doesn't scroll out to edge of the unobscured content.
Should we push this padding into an inner component?

Reply:
Yup! Added, had to do a bit amount of refactoring for the Article screen in the list/detail but it's looking good! Thanks